packages/calypso-e2e/src/lib/flows/start-site-flow.ts

```diff
@@ -43,20 +45,38 @@ export class StartSiteFlow {
 		await this.page.click( selectors.button( text ) );
 	}
 
+	/**
+	 * Enter site vertical.
+	 *
+	 * @param {string} vertical Name of the vertical to select
+	 */
+	async enterVertical( vertical: string ): Promise< void > {
+		await this.page.waitForLoadState( 'networkidle' );
+
+		const input = this.page.locator( selectors.verticalInput );
+		await input.fill( vertical );
+
+		const readBack = await input.inputValue();
+		if ( readBack !== vertical ) {
+			throw new Error( `Failed to set vertical: expected ${ vertical }, got ${ readBack }` );
+		}
+	}
+
 	/**
 	 * Enter blog name.
 	 *
 	 * @param {string} name Name for the blog.
 	 */
 	async enterBlogName( name: string ): Promise< void > {
 		await this.page.waitForLoadState( 'networkidle' );
-		// Wait for the input to be populated with the default value `Site Title`.
-		// See https://github.com/Automattic/wp-calypso/issues/64271.
-		const defaultInputlocator = this.page.locator( selectors.blogNameInput( 'Site Title' ) );
+		const defaultInputlocator = this.page.locator( selectors.blogNameInput );
+		// try waiting for element to become enabled
+		// we've had issues with this before https://github.com/Automattic/wp-calypso/issues/64271
+		waitForElementEnabled( this.page, selectors.blogNameInput );
```

Comment:
Since the input no longer has "Site Title" pre-filled, we can't use this selector anymore. This is similar to the old approach before https://github.com/Automattic/wp-calypso/issues/64271 where we waited for `enabled` state. I wasn't able to reproduce the issues myself, I think we'd better just see if they occur on CI. Open to other ideas for how to handle this 🤔

Reply:
The issue with the site name and tagline resetting is fixed in https://github.com/Automattic/wp-calypso/pull/64590.

Reply:
The above means you shouldn't need to use `waitForElementEnabled`. 

In fact I would consider `waitForElementEnabled` to be deprecated so would not encourage its use.

Alternatively can I suggest using locators?

```
const inputLocator = page.locator( selector );
await inputLocator.fill( name );
```

Reply:
Ahh great, I think I didn't understand the other comment and thought it was an issue purely related to the e2e tests.

So, I think I should add the `not(:disabled)` selectors back in as it was before https://github.com/Automattic/wp-calypso/pull/64458/files right?

Reply:
@roo2 

> So, I think I should add the not(:disabled) selectors back in as it was before

Right, the `input#siteTitle:not(:disabled)` selector should work right out of the box. The changes made in #64458 failed to make the script stable which was (as noted earlier) caused by a real bug.